source/logger.js

```diff
@@ -0,0 +1,60 @@
+if(Meteor.isServer)
+  winston = Npm.require('winston');
+
+Space.Logger = Space.Object.extend(Space, 'Logger', {
+
+  _logger: null,
+  _state: 'stopped',
+
+  Constructor: function(){
+
+    if(Meteor.isServer) {
+      this._logger = new (winston.Logger)({
+        transports: [
+          new (winston.transports.Console)()
+        ]
+      });
+    }
+    if(Meteor.isClient) {
+      this._logger = console
+    }
+  },
```

Comment:
Ah forget it – of course it needs to be more low-level, so that we can log low-level stuff! Or do you think it work otherwise too?